### DISCUSSION TITLE
希望早日支持 pokemonshowdown 汉化脚本

### DISCUSSION THREAD

**Discussion by l296499149** [shenruisi/Stay#21] [Unsupported userscript]
用途：将 pokemonshowdown 网站英文文本替换为中文

脚本链接：http://39.96.50.192:8000/avatars/static/20210709-chn-showdown.user.js

**Comment by shenruisi** [comment]
@l296499149 
请将 
// @require  http://libs.baidu.com/jquery/2.1.4/jquery.min.js
替换为
// @require  stay://vendor/jquery-3.5.1.min.js
后尝试
